style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 375px;
+  height: 5410px;
+}
+
+header .container {
+  width: 100%;
+  margin-bottom: 0;
+}
+
+header .container .second-container {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+}
+
+.icons div {
+  padding-right: 20px;
+}
+
+header .container .second-container div {
+  font-family: consolas, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+header .container .second-container nav {
+  padding-top: 5px;
+}
+
+.header-background-mobile {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 40px;
+}
+
+section {
+  background-image: url(image/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+  height: 700px;
+}
```

Comment:
- [ ] Kindly align the content of this section with flexbox; this gives us a lot of options to align all our elements, plus, it's responsive. 👍